examples/passport-oauth2-login/src/authentication-strategies/redirect-route.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {OperationObject, SchemasObject} from '@loopback/openapi-v3';
+import {ResolvedRoute, RouteEntry} from '@loopback/rest';
+import {RequestContext} from '@loopback/rest';
+import {
+  OperationArgs,
+  OperationRetval,
+  PathParameterValues,
+} from '@loopback/rest';
+
+export class RedirectRoute implements RouteEntry, ResolvedRoute {
+  // ResolvedRoute API
+  readonly pathParams: PathParameterValues = [];
+  readonly schemas: SchemasObject = {};
+
+  // RouteEntry implementation
+  readonly verb: string = 'get';
+  readonly path: string;
+  readonly spec: OperationObject = {
+    description: 'LoopBack Redirect route',
+    'x-visibility': 'undocumented',
+    responses: {},
+  };
+
+  constructor(
+    public readonly sourcePath: string,
+    public readonly targetLocation: string,
+    public readonly statusCode: number = 303,
+  ) {
+    this.path = sourcePath;
+  }
+
+  async invokeHandler(
+    {response}: RequestContext,
+    args: OperationArgs,
+  ): Promise<OperationRetval> {
+    response.redirect(this.statusCode, this.targetLocation);
+  }
+
+  updateBindings(requestContext: RequestContext) {
```

Comment:
Do we need this as an empty method? Please clarify. thx.

Reply:
agreed, removed